mobile-app/app/screens/AppNavigator/screens/Portfolio/components/AddressSelectionButtonV2.tsx

```diff
@@ -31,7 +32,7 @@ export function AddressSelectionButtonV2 (props: AddressSelectionButtonProps): J
       <ThemedTextV2
         ellipsizeMode='middle'
         numberOfLines={1}
-        style={[tailwind('text-sm font-semibold-v2 ml-2'), { minWidth: 10, maxWidth: 124 }]}
+        style={[tailwind('text-sm font-semibold-v2 ml-2'), { minWidth: 10, maxWidth: Platform.OS === 'android' ? 134 : 124 }]}
```

Comment:
still clipping when tested with this address `bcrt1q4tx5uhdu6wsmh084ht0zq5d0lhq2hxey0sdvx9`
![image](https://user-images.githubusercontent.com/40191153/183806719-3e2d34e8-20d8-4b39-afe3-c408d149541b.png)

Need your help to check if there is a better or more generic solution to address this issue?

Reply:
sure!